stdlib/TOML/src/print.jl

```diff
@@ -93,7 +93,7 @@ function printvalue(f::MbyFunc, io::IO, value::TOMLValue)
     value isa Dates.Time     ? Base.print(io, Dates.format(value, Dates.dateformat"HH:MM:SS.sss")) :
     value isa Dates.Date     ? Base.print(io, Dates.format(value, Dates.dateformat"YYYY-mm-dd")) :
     value isa Bool           ? Base.print(io, value ? "true" : "false") :
-    value isa Integer        ? Base.print(io, Int64(value)) :  # TOML specifies 64-bit signed long range for integer
+    value isa Integer        ? Base.show(io, value) :  # Julia's own printing should be compatible with TOML on integers
```

Comment:
Actually we should probably still print things "compactly":

```
julia> Base.show(UInt(10))
0x000000000000000a
```

better to print it without all the leading 0s.

Reply:
Yes, that was the point I was making: if we're going to parse hex values as `UInt64 then we need to print them without all the leading zeros. Always printing an even number of hex digits is common since two hex digits is a byte, but I don't feel too strongly about that.

Reply:
you get it, hex are printed in even numbers